config/gpt_template.json

```diff
@@ -0,0 +1,8 @@
+{
+    "model": "gpt-3.5-turbo",
```

Comment:
could you add a comment for a possible config for v4?

Reply:
I would just update it once the gpt-4 model is available and tested